examples/IBM.WatsonDeveloperCloud.VisualRecognition.v3.Example/VisualRecognitionServiceExample.cs

```diff
@@ -62,21 +62,21 @@ public VisualRecognitionServiceExample(string apikey)
             IsClassifierReady(_createdClassifierId);
             autoEvent.WaitOne();
             UpdateClassifier();
-            GetCollections();
-
-            DeleteDotnetCollections();
-
-            CreateCollection();
-            GetCollection();
-            GetCollectionImages();
-            AddCollectionImages();
-            GetCollectionImage();
-            GetCollectionImageMetadata();
-            AddCollectionImageMetadata();
-            DeleteCollectionImageMetadata();
-            FindSimilar();
-            DeleteCollectionImage();
-            DeleteCollection();
+            //GetCollections();

```

Comment:
remove the commented code.

Reply:
OK, I removed commented code and removed Collections from examples file!